base/logging.jl

```diff
@@ -311,12 +315,12 @@ function logmsg_code(_module, file, line, level, message, exs...)
             _module = $_module
             logger = current_logger_for_env(std_level, group, _module)
             if !(logger === nothing)
+                file = $file
```

Comment:
Why were these moved?

Reply:
Those values are needed to compute `id`.